tests/unit/test_checkpointing.py

```diff
@@ -267,8 +269,8 @@ def _test_checkpoint_fused_optimizer(args, model, hidden_dim, load_optimizer_sta
                               'deepspeed_adam'),
                          ])
 def test_checkpoint_zero_optimizer(tmpdir, zero_stage, use_cpu_offload, adam_optimizer):
-    if use_cpu_offload and not deepspeed.ops.__installed_ops__['cpu-adam']:
-        pytest.skip("cpu-adam is not installed")
+    #if use_cpu_offload and not deepspeed.ops.__installed_ops__['cpu-adam']:
+    #    pytest.skip("cpu-adam is not installed")
```

Comment:
these might be TODOs, just commenting so we remember there is some commented out skip logic.